client/src/components/ui/itemrow/PersonRowLocal.svelte

```diff
@@ -14,30 +15,31 @@
     export let name: User['name'];
     export let email: User['email'];
     export let picture: User['picture'];
-    export let globalPermission: User['permission'];
 
     // From staff.ts
     export let office: Staff['office'];
-    export let localPermission: Staff['permission'];
+    export let permission: Staff['permission'];
 
     const dispatch = createEventDispatcher();
     const rowEvent: PersonPayload = {
         ty: RowType.Person,
         id,
         office,
     };
+
+    const officeName = office ? $allOffices[office] : '';
```

Comment:
Same concerns as earlier. Isn't `office` already non-null?
```suggestion
    $: officeName = $allOffices[office] ?? '[No Office]';
```